DasharoPayloadPkg/BlSupportPei/BlSupportPei.c

```diff
@@ -342,7 +342,13 @@ MemInfoCallback (
         );
     } else if (Type == EFI_RESOURCE_SYSTEM_MEMORY) {
       if (Base < 0x100000000ULL) {
-        MemInfo->UsableLowMemTop = (UINT32)(Base + Size);
+        //
+        // Skip regions smaller than PeiMemSize as defined in BlPeiEntryPoint,

```

Comment:
```suggestion
        // Skip regions smaller than PeiMemSize as defined in BlPeiEntryPoint() below,
```
or define the size at the top.  I didn't get it's the entry point here at first because its name doesn't match the PEIM's name.

Reply:
Defined it on top, in case it needs to be adjusted it will have to be done in only one place: https://github.com/Dasharo/edk2/compare/b67eb08260c40f463eca88298e548cc70f8d3831..97938d04f1df2e564c1986fd34dc0dd00467cf54#diff-e11426c5786175fdf345eec361e1c716ab6792d8fddbd5b005e02ab589ad3d7bR11

Also changed some literal values to `BASE_1MB` and `BASE_4GB` for readability.